sysdig/resource_sysdig_secure_policy.go

```diff
@@ -107,6 +107,10 @@ func resourceSysdigSecurePolicy() *schema.Resource {
 										Required:         true,
 										ValidateDiagFunc: validateDiagFunc(validation.IntAtLeast(0)),
 									},
+									"name": {
+										Type:     schema.TypeString,
+										Required: true,
+									},
```

Comment:
By marking the new field as required, this is a breaking change for the users. They may not be expecting the need to add this kind of field. Either we make it optional, and provide a default value, or we release a new breaking Semver.